sample_file.py

```
@@ -0,0 +1,47 @@
+USERNAME = "admin2"
+PASSWORD = "password123"
+
+import os
+
+result = 0
+
+def infinite_loop():
+    while True:
+        print("This is an infinite loop")
+
+def sum_numbers(numbers):
```

Ruff

Missing return type annotation for public function `sum_numbers`

(ANN201)